.github/workflows/docs.yml

```diff
@@ -1,59 +1,71 @@
-name: docs
+name: Build and deploy docs
 
 on:
-    push:
-        branches:
-            - master
-            - docs-v2
-    workflow_dispatch:
+  push:
+    branches:
+      - master
+      - docs-v2
+  workflow_dispatch:
 
 jobs:
-    build:
-        if: ${{ !contains(github.event.head_commit.message, '[skip ci]') }}
-        runs-on: ubuntu-latest
-
-        steps:
-            -   uses: actions/checkout@v3
-                with:
-                    token: ${{ secrets.GH_TOKEN }}
```

Comment:
i am quite sure we need the personal access token, you won't be able to commit from the CI without it if the branch is protected (and maybe even without the branch protection)

Reply:
Oh, yeah, that makes sense. I will adjust it.

From which user is the `GH_TOKEN`, by the way?

Reply:
It's mine, feel free to replace with yours so you get notified about the docs build failures if you want.

Not sure if we have some GH user we could use to generate those tokens, but it needs to be personal access token, so I believe it needs to be connected to actual GH user account.

Reply:
We have @apify-service-account exactly for these usecases 🙂 It's already used here in the release workflow for triggering the docker image builds in https://github.com/apify/apify-actor-docker.

I'll make a new token in that user for this and use it here.

Reply:
Added the token, works fine now even with branch protection.